lib/modules/hosts/bime.js

```diff
@@ -0,0 +1,14 @@
+/* eslint no-unused-vars: [2, { "args": "none" }] */
+export default {
+	moduleID: 'bime',
+	name: 'Bime Analytics Dashboards',
+	domains: ['bime.io'],
+	logo: '//a.bime.io/assets/favicons/favicon.ico',
+	detect: ({ href }) => (/https?:\/\/(?:([^.]+)\.)bime\.io(?:\/([a-z0-9_-]+))+/i).exec(href),
```

Comment:
Nit: this non-capturing group: `(?:([^.]+)\.)` is redundant (i.e. use `([^.]+)\.`)